### ISSUE TITLE
[MP4Box] [DASH] [HLS] Add support for CMAF-compliant output

### ISSUE THREAD

**Issue by vladimir-kazakov** [gpac/gpac#1066]
There is a need to produce single content that can be consumed by both DASH and HLS clients. In this case, a content provider can create a single set of MP4 files that are referenced by different manifests for different adaptive streaming clients. In other words, HLS (.m3u8) is for Apple devices (since they don't support DASH), and DASH (.mpd) is for the rest of the world, but all manifests reference exactly the same .mp4 files.

CMAF (Common Media Application Format) is designed to make it possible. It's described in the [ISO/IEC 23000-19](https://www.iso.org/standard/71975.html) standard.

Currently (GPAC version 0.7.2-DEV-rev491-g0cad40bd-master), when creating dashed content, MP4Box output is not CMAF-compliant. This violates requirements of the [HLS standard](https://tools.ietf.org/html/rfc8216), mentioned in the [section 3.3](https://tools.ietf.org/html/rfc8216#section-3.3).

**Comment by jeanlf** [comment]
I suspect the issue is with cbcs encryption, we had a wrong parsing of the avc slice header now fixed on git. We are now patching windows build, the installers should soon be back online could you check them?

**Comment by vladimir-kazakov** [comment]
I tried GPAC version 0.7.2-DEV-rev534-g146aba3e-master. Now Safari plays such content, but there are a lot of blocking distortions on the video.

**Comment by jeanlf** [comment]
ok getting there. Is this always with the same 360p-750k.mp4 content?

**Comment by vladimir-kazakov** [comment]
Yes, the content is and was always the same (360p-750k.mp4). I tried GPAC version 0.7.2-DEV-rev535-gb052cb21-master. Blocking distortions are gone. Now everything plays correctly in Safari. Also checked playback on Android - blocking distortions disappeared there as well. Thanks for fixes!

But going back to the original issue - it still would be nice if MP4Box could output CMAF-compliant content.

**Comment by jeanlf** [comment]
now on git , I'll update the DASH / HLS howtos asap.
Feedback welcome :)
